src/components/ButtonPanel.js

```diff
@@ -1,41 +1,48 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 import '../App.css';
 
-function ButtonPanel() {
+const ButtonPanel = (props) => {
+  const { clickHandler } = props;
+
+
   return (
     <div id="button-panel" className="ButtonPanel">
       <div id="group-1">
-        <Button name="AC" color="lightgrey" />
-        <Button name="+/-" color="lightgrey" />
-        <Button name="%" color="lightgrey" />
-        <Button name="÷" />
+        <Button name="AC" color="lightgrey" clickHandler={clickHandler} />
+        <Button name="+/-" color="lightgrey" clickHandler={clickHandler} />
+        <Button name="%" color="lightgrey" clickHandler={clickHandler} />
+        <Button name="÷" clickHandler={clickHandler} />
       </div>
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.